controllers/complianceeventsapi/migrations/000001_compliance_history_initial_tables.up.sql

```diff
@@ -12,63 +12,55 @@ CREATE INDEX IF NOT EXISTS idx_clusters_name ON clusters (name);
 CREATE TABLE IF NOT EXISTS parent_policies(
    id serial PRIMARY KEY,
    name TEXT NOT NULL,
+   namespace TEXT NOT NULL,
    categories TEXT [],
    controls TEXT [],
    standards TEXT [],
    UNIQUE (name, categories, controls, standards)
```

Comment:
Does this also need `namespace`?

Reply:
Yeah, my thought is that OCM policies in the `policies` table won't set the namespace because it's always the cluster namespace to prevent data duplication. The root/parent policy namespace will be a separate column rather than the original plan of `<namespace>.<name>`. This will make filtering easier and allow us to have namespaced based RBAC for viewing compliance events in the future if desired.

Reply:
I've updated the design PR to reflect that:
https://github.com/open-cluster-management-io/enhancements/pull/105

Reply:
Good catch! Thanks!